coding-agent/fix_issue.py

```
@@ -0,0 +1,163 @@
+        cwd = os.path.abspath(repo_dir)
+
+        # 2. Get Issue Details
+        print("Fetching issue details...")
+        issue_body = run_command(f"gh issue view {issue_num} --json title,body --template 'Title: {{{{ .title }}}}\n\n{{{{ .body }}}}'", cwd=cwd)
+        
+        # 3. Create Branch
+        branch_name = f"fix/issue-{issue_num}"
+        print(f"Creating branch {branch_name}...")
+        try:
+            run_command(f"git checkout -b {branch_name}", cwd=cwd)
+        except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)